vmdb/db/migrate/20150111161446_create_nodes.rb

```diff
@@ -3,7 +3,7 @@ def up
     create_table :container_nodes do |t|
       t.string    :ems_ref
       t.string    :name
-      t.timestamp  :creation_timestamp
+      t.timestamp  :creation_timestamp, :null => true
```

Comment:
Note, even with my changes in this PR, I still get a few warnings of the same kind that I haven't tracked down yet.  Perhaps datetime or other methods raise the warning but mention timestamp.